core/shared/src/test/scala/scalaz/zio/BaseCrossPlatformSpec.scala

```diff
@@ -46,4 +49,14 @@ abstract class BaseCrossPlatformSpec extends Specification with DefaultRuntime {
     unsafeRunToFuture(io.sandbox.mapError(FiberFailure(_))).map(p.success)
     p.future
   }
+
+  implicit class ZIOMustExpectable[R, E, A](zio: ZIO[R, E, A]) {
```

Comment:
How about making `zio` a private val and the class extend `AnyVal`?

Reply:
Actually `Value classes can have only one non-private val parameter`.
I would leave as it is. `zio` is currently private (no `val`) and both methods are available without any extra imports (like it would be in case of value class, because it cannot be placed inside class `BaseCrossPlatformSpec`).